wiki/Contests/en.md

```diff
@@ -2,291 +2,420 @@
+| [#8](https://osu.ppy.sh/home/news/2015-02-11-monthly-beatmapping-contest-8) | raja - the light | ![osu!][osu!] | ![][flag_DE] [Damnae](https://osu.ppy.sh/users/989377) | ![][flag_FR] [Cherry Blossom](https://osu.ppy.sh/users/1156742) | ![][flag_RU] [Shadren](https://osu.ppy.sh/users/745494) and ![][flag_CN] [Frey](https://osu.ppy.sh/users/695678) | [Results](https://osu.ppy.sh/home/news/2015-04-08-monthly-beatmapping-contest-8-results) |
+| [#9](https://osu.ppy.sh/home/news/2015-04-13-monthly-beatmapping-contest-9-taiko-only) | Furries in a Blender - Storm World | ![osu!taiko][osu!taiko] | ![][flag_TW] [Firce777](https://osu.ppy.sh/users/274072) | ![][flag_SE] [Hanjamon](https://osu.ppy.sh/users/1703330) and ![][flag_JP] [Naryuga](https://osu.ppy.sh/users/522188) | ![][flag_US] [MMzz](https://osu.ppy.sh/users/128993) | [Results](https://osu.ppy.sh/home/news/2015-06-27-monthly-beatmapping-contest-9-results) |
+| [#10](https://osu.ppy.sh/home/news/2015-06-15-monthly-beatmapping-contest-10-ctb-only) | Hylian Lemon - Foresight Is For Losers | ![osu!catch][osu!catch] | ![][flag_CL] [ZiRoX](https://osu.ppy.sh/users/200768) | ![][flag_GB] [WildOne94](https://osu.ppy.sh/users/3482692) | ![][flag_ID] [Xinely](https://osu.ppy.sh/users/1521445) | [Results](https://osu.ppy.sh/home/news/2015-08-06-monthly-beatmap-contest-10-results-ctb) |
+| [#11](https://osu.ppy.sh/home/news/2015-08-22-monthly-beatmapping-contest-11-mania-only) | Freedom Planet - Dragon Valley (Toni Leys Remix feat. Esteban Bellucci) | ![osu!mania][osu!mania] | ![][flag_ID] [LordRaika](https://osu.ppy.sh/users/3265023) | ![][flag_VN] [Kuo Kyoka](https://osu.ppy.sh/users/2596306) | ![][flag_ID] [Shionia-Chan](https://osu.ppy.sh/users/7015183) | [Results](https://osu.ppy.sh/home/news/2015-12-03-monthly-beatmapping-contest-11-results) |
+| [#12](https://osu.ppy.sh/home/news/2016-03-04-beatmapping-contest-12-osu) | IAHN - Candy Luv | ![osu!][osu!] | ![][flag_KR] [Taeyang](https://osu.ppy.sh/users/2732340) | ![][flag_CA] [Monstrata](https://osu.ppy.sh/users/2706438) | ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) | [Results](https://osu.ppy.sh/home/news/2016-05-26-monthly-beatmapping-contest-12-results) |
+| [#13](https://osu.ppy.sh/home/news/2016-12-09-monthly-beatmapping-contest-13-now-open) | BilliumMoto - HDHR | ![osu!][osu!] | ![][flag_MY] [RyoKazuka](https://osu.ppy.sh/users/6258586) | ![][flag_ID] [Fort](https://osu.ppy.sh/users/2805457) | ![][flag_DE] [Pho](https://osu.ppy.sh/users/3624692) | [Results](https://osu.ppy.sh/home/news/2017-04-02-monthly-beatmapping-contest-13-results) |
+
+After a long hiatus, the Monthly Beatmapping Contest is being rebooted in January 2020 with a brand new schedule, ruleset, voting procedure, and reward scheme. Many aspects of the competition have been dramatically altered to make way for a more engaging and attractive contest, in which some of them are as follows:
```

Comment:
```suggestion
After a long hiatus, the Monthly Beatmapping Contest was rebooted in January 2020 with a brand new schedule, rule set, voting procedure, and reward scheme. Many aspects of the competition were dramatically altered to make way for a more engaging and attractive contest:
```
may be worth mentioning when that hiatus was

Reply:
think i'll keep the "in which some of them are as follows:" here as the list only covers some of the main changes (or else the list would be unnecessarily long for what is supposed to be an index page of some sort lol)

applied everything else otherwise